client/my-sites/invites/invite-accept-logged-out/index.jsx

```diff
@@ -52,6 +52,10 @@ class InviteAcceptLoggedOut extends Component {
 
 	submitForm = ( form, userData, _, afterSubmitCallback = noop ) => {
 		const { invite } = this.props;
+		recordTracksEvent( 'calypso_invite_accept_logged_out_submit', {
```

Comment:
I am not sure if this function will be called when using a passwordless form. I believe we should have something like that [here](https://github.com/Automattic/wp-calypso/blob/trunk/client/blocks/signup-form/passwordless.jsx#L173).
Can we put this PR on hold for a while? I am working on the passwordless invite and once we have a clear idea on how to achieve that, we can revisit this one. WDYT?